lib/sync/installation.js

```diff
@@ -37,6 +37,12 @@ module.exports.processInstallation = (app, queues) => {
     // Get a fresh subscription instance
     const subscription = await Subscription.getSingleInstallation(jiraHost, installationId);
 
+    // handle promise rejection when an org is removed during a sync
+    if (subscription == null) {
+      console.log('Organization has been deleted. Other active syncs will continue.');
```

Comment:
This might be enhanced by using `app.log` instead of `console.log` as the App logger has extra context in it and will follow the json output patterns, generally we try to avoid `console.log` within the application.